tests/testVisitInfo.py

```diff
@@ -33,10 +33,10 @@
 import lsst.afw.image as afwImage
 
 RotTypeEnumNameDict = {
-    afwImage.RotType_UNKNOWN: "UNKNOWN",
-    afwImage.RotType_SKY: "SKY",
-    afwImage.RotType_HORIZON: "HORIZON",
-    afwImage.RotType_MOUNT: "MOUNT",
+    afwImage.RotType.UNKNOWN: "UNKNOWN",
+    afwImage.RotType.SKY: "SKY",
+    afwImage.RotType.HORIZON: "HORIZON",
+    afwImage.RotType.MOUNT: "MOUNT",
```

Comment:
I wonder what we should do with this. One the one hand I prefer using `.` and this should definitely be the future. However, I wonder how much code expects `_` to also exist. Perhaps it is better to enforce this sooner rather than later, but it is an API change (one that we have made in a number of places already for sure though).